ci-operator/step-registry/baremetal/lab/agent/conf/static/baremetal-lab-agent-conf-static-commands.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+RENDEZVOUS_IP="$(yq -r e -o=j -I=0 ".[0].ip" "${SHARED_DIR}/hosts.yaml")"
+
+cat > "${SHARED_DIR}/agent-config.yaml" <<EOF
+apiVersion: v1beta1
+kind: AgentConfig
+rendezvousIP: ${RENDEZVOUS_IP}
+additionalNTPSources:
+- ${AUX_HOST}
+hosts: []
+EOF
+
+# shellcheck disable=SC2154
+for bmhost in $(yq e -o=j -I=0 '.[]' "${SHARED_DIR}/hosts.yaml"); do
+  # shellcheck disable=SC1090
+  . <(echo "$bmhost" | yq e 'to_entries | .[] | (.key + "=\"" + .value + "\"")')
+  ADAPTED_YAML="
+  hostname: ${name}
+  role: ${name%%-[0-9]*}
+  rootDeviceHints:
+    ${root_device:+deviceName: ${root_device}}
+    ${root_dev_hctl:+hctl: ${root_dev_hctl}}
+  interfaces:
+  - macAddress: ${mac}
+    name: ${baremetal_iface}
+  networkConfig:
+    interfaces:
+    - name: ${baremetal_iface}
+      type: ethernet
+      state: up
+      ipv4:
+        enabled: ${ipv4_enabled}
+        dhcp: false
+        address:
+            - ip: ${ip}
+              prefix-length: 23
+      ipv6:
+        enabled: ${ipv6_enabled}
+"
+
+  # split the ipi_disabled_ifaces semi-comma separated list into an array
+  IFS=';' read -r -a ipi_disabled_ifaces <<< "${ipi_disabled_ifaces}"
+  for iface in "${ipi_disabled_ifaces[@]}"; do
+    # Take care of the indentation when adding the disabled interfaces to the above yaml
+    ADAPTED_YAML+="
+    - name: ${iface}
+      type: ethernet
+      state: up
+      ipv4:
+        enabled: false
+        dhcp: false
+      ipv6:
+        enabled: false
+        dhcp: false
+    "
+  done
+  
+  # Take care of the indentation when adding the routes to the above yaml
+  # TODO check what happens with multiple baremetal_ifaces, we are not looping here
```

Comment:
You'd remove this todo, interfaces are under loops as needed. baremetal_iface is expected to be the one interface that provides outbound networks reachability. An "additional networks" handling can come later if there are scenarios requiring it, but that would be another feature.

Reply:
removed